docs/known-issues.md

```diff
@@ -12,16 +12,186 @@ The workaround is to edit the migration code generated by EF Core to make it wor
 Github issues: https://github.com/cockroachdb/cockroach/issues/112090
 
 
-CockroachDB doesn’t support the function pg_terminate_backend(). EnsureDeleted and EnsureDeletedAsync do not work.
+### CockroachDB doesn’t support the function pg_terminate_backend(). EnsureDeleted and EnsureDeletedAsync do not work.
 
-This code
+This code will not work.
 
 ```csharp
 await using var ctx = new BlogContext();
 await ctx.Database.EnsureDeletedAsync();
 await ctx.Database.EnsureCreatedAsync();
 ```
 
+### CockroachDB doesn't support concurrency token column
+Github issue: https://github.com/cockroachdb/cockroach/issues/33296
 
+Entity Framework Core supports the concept of optimistic concurrency - a property on your entity is designated as a concurrency token, and EF Core detects concurrent modifications by checking whether that token has changed since the entity was read.
+
+In PostgreSQL, the xmin column, which xmin holds the ID of the latest updating transaction, can be used as concurrency token column.
+
+### CockroachDB doesn’t support such a column.
+
+This code will not work as with PostgreSQL or other databases
+
+```csharp
+class MyContext : DbContext
+{
+    public DbSet<SomeEntity> SomeEntities { get; set; }
+
+    protected override void OnModelCreating(ModelBuilder modelBuilder)
+    {
+        modelBuilder.Entity<SomeEntity>()
+            .Property(b => b.Version)
+            .IsRowVersion();
+    }
+}
+```
+
+### CockroachDB doesn't support date range types
+Github issue: https://github.com/cockroachdb/cockroach/issues/27791
+
+This code will not work
+
+```csharp
+public class Entity
+{
+        public NpgsqlRange<DateTime> TimestamptzDateTimeRange { get; set; }
+       
+        [Column(TypeName = "tsrange")]
+        public NpgsqlRange<DateTime> TimestampDateTimeRange { get; set; }
+}
+```
+
+### CockroachDB uses 64-bit signed integers for INT by default
+efcore.pg follows PostgreSQL default where 32-bit integers are used.
+
+Workaround: Change the cluster setting default_int_size to 4
+
+SET CLUSTER SETTING sql.defaults.default_int_size = 4;
+
+CockroachDB uses rowid for SERIAL column by default
+This conflicts with how efcore.pg handles SERIAL columns
+
+Workaround: Change the cluster setting serial_normalization to use sql_sequence, similar to PostgreSQL
```

Comment:
`serial_normalization` is a session variable, not a cluster setting

Reply:
`sql.defaults.serial_normalization` is the cluster setting.

but we recommend using `ALTER ROLE ALL SET serial_normalization = ...` instead. same goes for default_int_size above